domains/dscbmr.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "RAYXSTYDIOZ",
+    "email": "syedyaseeralirayan@gmail.com", 
+    "discord":  "1130109258017550357"
+  },
+  "record": {
+    "TXT": "dh=af0561dc7d2f7eb0ef153cb3a270c0b98fe5c36e"
```

Comment:
Not the way to connect your domain to Discord, check here https://www.is-a.dev/docs/discord_verification/